### DISCUSSION TITLE
Problems with soup

### DISCUSSION THREAD

**Discussion by N8TB15** [Security-Onion-Solutions/securityonion#8261] [General] [answered]
Solution to soup not verifying iso:

Change iso name specifically to: 
(Do not use iso name as it was downloaded)

securityonion.iso

Should upgrade from there.

**Comment by dougburks** [comment, marked as answer]
I'm not sure I understand. You shouldn't need to rename the ISO image.

**Reply by N8TB15** [reply]
Doug,

I was upgrading on air gapped network.  Each time I ran soup it would error out at about 20 minutes saying it couldn't find iso verify its there.  I don't have the exact message at the moment but I can check logs later and provide them.  The moment I matched the iso file as it was in documentation it went through no issue.  

After multiple attempts changing the file name to securityonion.iso it functioned properly.  It was initially the full name of newest iso.

**Reply by dougburks** [reply]
Were you running `soup` and then letting it prompt you for the path to the ISO image OR were you copying the command `sudo soup -y -f /home/user/securityonion.iso` straight from the documentation? If the latter, that's meant as an example and you should change that command to reflect the actual name of the downloaded ISO image.

**Reply by N8TB15** [reply]
I provided the iso path.

**Reply by N8TB15** [reply]
With the proper name of iso not directly from documentation.

**Reply by N8TB15** [reply]
I ran this multiple times only to error out, even after changing permissions:

sudo soup -y -f /home/user/securityonion-2.3.130-20220607.iso

It did not function properly until I renamed iso and ran:

sudo soup -y -f /home/user/securityonion.iso


I will provide the log message later today.  I posted this as searching the error brought me to posts saying do fresh install to complete the upgrade.  I want to help save others time, rather than do a fresh install.

**Reply by N8TB15** [reply]
![20220711_114102.jpg](https://user-images.githubusercontent.com/79382921/178302866-7688d2c6-54f0-4089-8019-f10033decb06.jpg)

**Reply by dougburks** [reply]
Is your user account literally named `user`?

**Reply by N8TB15** [reply]
No, I Created a user specifically for upgrade.  I Created a user with proper permissions to allow for upload via WinSCP. I used the full path properly not sure why it did it that way in log.

**Reply by dougburks** [reply]
I normally just scp the file to my normal user account and soup upgrades fine without having to rename the file. I wonder if there is something strange with your `user` account or perhaps the SELinux context of its directory.

**Reply by N8TB15** [reply]
I'm not sure.  I found it weird.  Thought the script could have really strict input parameters.  But as I mentioned each of my errors led me to posts saying do a rebuild and that worked.   I'll attempt to recreate it in a lab environment and see if I get the same issue and update you.
